### ISSUE TITLE
The `indentation` rule never measures a comment behind the last bodiless at-rule of a stylesheet, which PostCSS files into the root's `raws.after`

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#592]
### What minimal example or steps are needed to reproduce the bug?

Where the last node of a stylesheet is an at-rule carrying neither a block nor a semicolon of its own, that at-rule runs to the end of the file, and PostCSS files a comment written behind it into the root's `raws.after` rather than into a node of its own — `atrule()` sees the end of the file, hands the at-rule's `raws.between` over to `this.spaces`, and `endFile()` puts that in the root's raw. Inside a block the same comment goes into the at-rule's `raws.between`, which `indentation` measures as a line of the block since #510; at the root it goes into a raw nobody measures.

```css
@import "x"
	/* c */
```

Under `"tab"` that draws nothing, and `--fix` leaves it as it is, where the same comment behind a semicolon, or behind a rule, is a comment node of the root and is reported at `2:3` as `Expected indentation of 0 tabs`:

| input | syntax | warnings |
| --- | --- | --- |
| `@import "x"⏎\t/* c */⏎` | css, scss, less | none |
| `a {}⏎@import "x"⏎\t\t/* c */⏎` | css | none |
| `@import "x";⏎\t\t/* c */⏎` | css | `2:3 Expected indentation of 0 tabs` |
| `a {}⏎\t\t/* c */⏎` | css | `2:3 Expected indentation of 0 tabs` |
| `<style>⏎@import "x"⏎\t/* c */⏎</style>⏎` | html | none |
| `<a style="@import 'x'⏎\t/* c */"></a>⏎` | html | none |
| a template holding `@extend .b` and `/* c */` a level deeper below it | styled | none |
| a template holding `color: red;` and `/* c */` a level deeper below it | styled | `3:3 Expected indentation of 1 tab` |

Measured on `b48ef4f`, the branch of #510, and the same on `177f15b`, the commit it stands on: the comment is in the root's raw on both.

### What minimal configuration is needed to reproduce the bug?

```json
{
	"plugins": ["@stylistic/stylelint-plugin"],
	"rules": {
		"@stylistic/indentation": "tab"
	}
}
```

### What did you expect to happen?

A comment standing on a line of its own behind the last statement of a stylesheet is measured like a comment behind any other statement — at the level of the root it stands in — wherever the parser filed it.

### What actually happened?

The line is measured by nobody: the at-rule's `raws.between` is empty, so the parameters' reading has nothing behind them, and the root's `raws.after` is a raw the rule never reads for indentation.

### Notes

Found while closing #510, which settled whose line a comment a bodiless at-rule swallowed is — the block's — inside a block, where the parser files it into `raws.between`. This is the same comment at the end of a file, of a `<style>` element, of a styled template and of a `style` attribute, where the parser files it into the root instead. The root of an inline `style` attribute holds nothing but such statements, so every trailing comment there goes unmeasured.
